Lib/asyncio/base_events.py

```diff
@@ -1857,12 +1857,10 @@ def call_exception_handler(self, context):
                                  exc_info=True)
 
     def _add_callback(self, handle):
-        """Add a Handle to _scheduled (TimerHandle) or _ready."""
+        """Add a Handle to _ready."""
         assert isinstance(handle, events.Handle), 'A Handle is required here'
```

Comment:
I wonder if we should drop this assertion as well? It's still going to be the slowest part of the method.

Reply:
Removed in 7ee5f56e9b8ce8329ff3387314848affb783b83f